### PR TITLE
[Communication]filtering generated classes for code coverage

### PR DESCRIPTION
**Pull request by paolamvhz**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs [hunk 1]
```diff
@@ -15655,15 +15655,15 @@ public static partial class BlobsModelFactory
         /// Creates a new AccountInfo instance for mocking.
         /// </summary>
         public static AccountInfo AccountInfo(
-            Azure.Storage.Blobs.Models.SkuName skuName,
+            bool isHierarchicalNamespaceEnabled,
             Azure.Storage.Blobs.Models.AccountKind accountKind,
-            bool isHierarchicalNamespaceEnabled)
+            Azure.Storage.Blobs.Models.SkuName skuName)
```
Comment (kasobol-msft) [inline sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs:15660]: Why did you change Storage's generated clients?
Btw. some of these changes might be breaking.
FYI @seanmcc-msft

**Comment by jsquire** [comment]
Hi @paolamvhz.  There hasn't been recent engagement on this PR.  Would you please be so kind as to let us know if this is still an active work stream by removing the `no-recent-activity` label?  Otherwise, we'll close this out in 7 days.

**Comment by jsquire** [comment]
Since there hasn't been recent engagement, this is being closed out.  Feel free to reopen if you'd like to continue working on these changes.  Please be sure to remove the `no-recent-activity` label otherwise, this will be closed again in the next cleanup pass.
